### PR TITLE
[Tokenizers] package metadata and minor fixes

### PR DESCRIPTION
**Pull request by mryzhov**
- Updated readme and included it to the package
- Removed unused CUSTOM_OP_LIB environment variable from the pipelines
- changed the name of the python package

### CODE REVIEW

File: modules/custom_operations/user_ie_extensions/tokenizer/python/README.md [hunk 1]
```diff
@@ -6,40 +6,32 @@
...
+3. Go to `modules/custom_operations` and run:
 ```bash
 # to use converted tokenizers or models combined with tokenizers
 pip install .
 # to convert tokenizers from transformers library
-pip install .[transformers] 
+pip install .[transformers]
 # for development and testing the library
-pip isntall -e .[all]
+pip istall -e .[all]
```
Comment (apaniukov) [inline modules/custom_operations/user_ie_extensions/tokenizer/python/README.md:26]:
```suggestion
pip install -e .[all]
```

File: modules/custom_operations/user_ie_extensions/tokenizer/python/README.md [hunk 2]
```diff
@@ -6,40 +6,32 @@
...
 ### Convert HuggingFace tokenizer
 
 ```python
 from transformers import AutoTokenizer
 from openvino import compile_model
-from ov_tokenizer import init_extension, convert_tokenizer, pack_strings
-
-
-init_extension("path/to/libuser_ov_extensions.so")
+from ov_tokenizer import convert_tokenizer, pack_strings
```
Comment (apaniukov) [inline modules/custom_operations/user_ie_extensions/tokenizer/python/README.md:34]: `ov_tokenizer` should be renamed to `openvino_tokenizer` too

File: modules/custom_operations/user_ie_extensions/tokenizer/python/README.md [hunk 3]
```diff
@@ -123,14 +111,10 @@ from transformers import AutoModelForCausalLM, AutoTokenizer
 from ov_tokenizer import (
     add_greedy_decoding,
     convert_tokenizer,
-    init_extension,
     pack_strings,
     unpack_strings,
 )
 
-
-init_extension("path/to/libuser_ov_extensions.so")
```
Comment (ilya-lavrenov) [inline modules/custom_operations/user_ie_extensions/tokenizer/python/README.md:132]: looks like we need to remove `init_extension` at all?

File: modules/custom_operations/user_ie_extensions/tokenizer/python/README.md (hunk 2 above)
Comment (ilya-lavrenov) [reply modules/custom_operations/user_ie_extensions/tokenizer/python/README.md:34]: and we need to rename the folder itself https://github.com/openvinotoolkit/openvino_contrib/tree/e4909171e279fd792ae609d4d15e5dbc0b0513ad/modules/custom_operations/user_ie_extensions/tokenizer/python/ov_tokenizer

File: modules/custom_operations/user_ie_extensions/tokenizer/python/README.md (hunk 3 above)
Comment (ilya-lavrenov) [reply modules/custom_operations/user_ie_extensions/tokenizer/python/README.md:132]: that PR is merged, let's remove `init_extension` from everywhere

File: .ci/azure/linux.yml [hunk 4]
```diff
@@ -200,6 +199,8 @@ jobs:
       source $(WORK_DIR)/.env3/bin/activate
       # need to enable sparse_conv tests with new Open3D release
       python -m pytest -k "not sparse_conv" tests/run_tests.py
+    variables:
+      CUSTOM_OP_LIB: $(BIN_DIR)/libuser_ov_extensions.so
```
Comment (apaniukov) [inline .ci/azure/linux.yml:203]: image

File: .ci/azure/windows.yml [hunk 5]
```diff
@@ -197,6 +196,8 @@ jobs:
       call C:\tools\opencv\build\setup_vars_opencv4.cmd
       call $(SETUPVARS) -pyver 3.8 && ^
       $(PYTHON_EXE) -m pytest -k "not sparse_conv" tests\run_tests.py
+    variables:
+        CUSTOM_OP_LIB: $(BUILD_DIR_CONTRIB)\user_ie_extensions\user_ov_extensions.dll
```
Comment (apaniukov) [inline .ci/azure/windows.yml:200]: image

**Review by apaniukov — APPROVED** [review]
Merge after CI fixes.

File: modules/custom_operations/user_ie_extensions/tokenizer/python/README.md (hunk 2 above)
Comment (mryzhov) [reply modules/custom_operations/user_ie_extensions/tokenizer/python/README.md]: renamed to openvino_tokenizers

**Review by ilya-lavrenov — APPROVED** [review]

File: .ci/azure/mac.yml [hunk 6]
```diff
@@ -163,22 +163,26 @@ jobs:
...
+      # Find and install wheel
+      pushd $(INSTALL_DIR)/tools
+        wheel_name=$(find . -name 'openvino-*.whl')
+        python -m pip install $wheel_name
+      popd
     workingDirectory: $(WORK_DIR)
     displayName: 'Create virtual env'
 
   - script: |
       source .venv/bin/activate && source $(SETUPVARS)
```
Comment (ilya-lavrenov) [inline .ci/azure/mac.yml:175]:
```suggestion
      source venv/bin/activate && source $(SETUPVARS)
```

File: .ci/azure/mac.yml [hunk 7]
```diff
@@ -163,22 +163,26 @@ jobs:
...
       python -m pip install build
-      python -m build --wheel --outdir $(BUILD_WHEEL_DIR) $(REPO_DIR)\modules\custom_operations
+      python -m build --wheel --outdir $(BUILD_WHEEL_DIR) $(REPO_DIR)/modules/custom_operations
     workingDirectory: $(WORK_DIR)
     displayName: 'Build tokenizers wheel'
 
   - script: |
       source .venv/bin/activate
```
Comment (ilya-lavrenov) [inline .ci/azure/mac.yml:182]:
```suggestion
      source venv/bin/activate
```

File: .ci/azure/mac.yml [hunk 8]
```diff
@@ -163,22 +163,26 @@ jobs:
...
+      pushd $(INSTALL_DIR)/tools
+        wheel_name=$(find . -name 'openvino-*.whl')
+        python -m pip install $wheel_name
+      popd
     workingDirectory: $(WORK_DIR)
     displayName: 'Create virtual env'
 
   - script: |
-      source .venv/bin/activate && source $(SETUPVARS)
+      source $(WORK_DIR)venv/bin/activate && source $(SETUPVARS)
```
Comment (ilya-lavrenov) [inline .ci/azure/mac.yml:175]:
```suggestion
      source $(WORK_DIR)/venv/bin/activate && source $(SETUPVARS)
```

**Pull request merged**